reactive/spark.py

```diff
@@ -123,5 +119,3 @@ def client_present(client):
 @when_not('spark.started')
 def client_should_stop(client):
     client.clear_spark_started()
-
-
```

Comment:
Unit testing would involve a lot of fairly clunky mocks, so I understand relying on integration testing for the charms in general. A linter is probably a better tool for catching the sorts of typos and minor mistakes that a heavily mocked unit test would catch, anyway.